jupyterlab/extensions/pypi.py

```diff
@@ -31,30 +35,60 @@
 )
 
 
-async def _fetch_package_metadata(name: str, latest_version: str, base_url: str) -> dict:
-    http_client = tornado.httpclient.AsyncHTTPClient()
-    response = await http_client.fetch(
-        base_url + f"/{name}/{latest_version}/json",
-        headers={"Content-Type": "application/json"},
-    )
-    data = json.loads(response.body).get("info")
-
-    # Keep minimal information to limit cache size
-    return {
-        k: data.get(k)
-        for k in [
-            "author",
-            "bugtrack_url",
-            "docs_url",
-            "home_page",
-            "license",
-            "package_url",
-            "project_url",
-            "project_urls",
-            "summary",
-        ]
+class ProxiedTransport(xmlrpc.client.Transport):
+    def set_proxy(self, host, port=None, headers=None):
+        self.proxy = host, port
+        self.proxy_headers = headers
+
+    def make_connection(self, host):
+        connection = http.client.HTTPConnection(*self.proxy)
+        connection.set_tunnel(host, headers=self.proxy_headers)
+        self._connection = host, connection
+        return connection
+
+
+xmlrpc_transport_override = None
+
+http_proxy_url = environ.get('http_proxy') or environ.get('HTTP_PROXY')
+proxies = {}
```

Comment:
Let's use the default httpx value:

```suggestion
proxies = None
```